dojo/api_v2/views.py

```diff
@@ -2536,116 +3588,128 @@ def report_generate(request, obj, options):
-        report_subtitle = str(product)
-        findings = ReportFindingFilter(request.GET, product=product, queryset=prefetch_related_findings_for_report(Finding.objects.filter(
-            test__engagement__product=product)))
+        str(product)
+        findings = ReportFindingFilter(
+            request.GET,
+            product=product,
+            queryset=prefetch_related_findings_for_report(
+                Finding.objects.filter(test__engagement__product=product)
+            ),
+        )
         ids = set(finding.id for finding in findings.qs)
-        engagements = Engagement.objects.filter(test__finding__id__in=ids).distinct()
+        engagements = Engagement.objects.filter(
+            test__finding__id__in=ids
+        ).distinct()
         tests = Test.objects.filter(finding__id__in=ids).distinct()
-        ids = get_endpoint_ids(Endpoint.objects.filter(product=product).distinct())
+        ids = get_endpoint_ids(
+            Endpoint.objects.filter(product=product).distinct()
+        )
         endpoints = Endpoint.objects.filter(id__in=ids)
 
     elif type(obj).__name__ == "Engagement":
         engagement = obj
-        findings = ReportFindingFilter(request.GET, engagement=engagement,
-                                       queryset=prefetch_related_findings_for_report(Finding.objects.filter(test__engagement=engagement)))
+        findings = ReportFindingFilter(
+            request.GET,
+            engagement=engagement,
+            queryset=prefetch_related_findings_for_report(
+                Finding.objects.filter(test__engagement=engagement)
+            ),
+        )
         report_name = "Engagement Report: " + str(engagement)
 
-        report_title = "Engagement Report"
-        report_subtitle = str(engagement)
+        str(engagement)
 
         ids = set(finding.id for finding in findings.qs)
         tests = Test.objects.filter(finding__id__in=ids).distinct()
-        ids = get_endpoint_ids(Endpoint.objects.filter(product=engagement.product).distinct())
+        ids = get_endpoint_ids(
+            Endpoint.objects.filter(product=engagement.product).distinct()
+        )
         endpoints = Endpoint.objects.filter(id__in=ids)
 
     elif type(obj).__name__ == "Test":
         test = obj
-        findings = ReportFindingFilter(request.GET, engagement=test.engagement,
-                                       queryset=prefetch_related_findings_for_report(Finding.objects.filter(test=test)))
-        filename = "test_finding_report.pdf"
-        template = "dojo/test_pdf_report.html"
+        findings = ReportFindingFilter(
+            request.GET,
+            engagement=test.engagement,
+            queryset=prefetch_related_findings_for_report(
+                Finding.objects.filter(test=test)
+            ),
+        )
         report_name = "Test Report: " + str(test)
-        report_title = "Test Report"
-        report_subtitle = str(test)
+        str(test)
```

Comment:
```suggestion

```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/3a2c9d288fe5aec61c4cc0739853f148a75e04bb